akka-stream/src/main/scala/akka/stream/javadsl/Tcp.scala

```diff
@@ -144,6 +144,17 @@ class Tcp(system: ExtendedActorSystem) extends akka.actor.Extension {
       .map(new IncomingConnection(_))
       .mapMaterializedValue(_.map(new ServerBinding(_))(ec).toJava))
 
+  @Deprecated
```

Comment:
ah, this is the java annotation, hm I think we use the scala annotation for such things (I hope that will show up as deprecated when used from java)